bin/oneoff/backfill_data/school_info_id

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+
+# This script copies user_id-school_info_id pairs from the pd_enrollments
+# table into the school_info_id column in users.
+
+require_relative '../../../dashboard/config/environment'
+
+# build hash mapping school_info_id to school_id
+school_info_id_school_id_mapping = {}
+
+batch_number = 0
+SchoolInfo.find_in_batches(batch_size: 2500) do |batch|
+  puts "PROCESSING SCHOOL_INFO: #{batch_number}..."
+  batch.each do |school_info|
+    school_info_id_school_id_mapping[school_info.id] = school_info.school_id
+  end
+  puts "PROCESSED SCHOOL_INFO: #{batch_number}..."
+  batch_number += 1
+end
```

Comment:
note if we don't care about the batch number, we could use the shorthand [find_each](https://apidock.com/rails/ActiveRecord/Batches/find_each).